lms/static/certificates/sass/views/_validate.scss

```diff
@@ -0,0 +1,288 @@
+// edX: Certificates - View: DIY Validation
```

Comment:
@marcotuts, do we need this view/feature? If not, I'll remove this styling partial.